mmv1/templates/terraform/decoders/vertex_model.go.erb

```diff
@@ -0,0 +1,7 @@
+// Model's operation response puts the resource name under "model" instead of "name".
```

Comment:
Decoders run on the GET response, not the operation response. GET should return a Model object per https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.models/get#response-body which *does* have a `name` field. Am I missing something here?

Reply:
They also run on the operation response: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/resource.erb#L295